bindgen-integration/build.rs

```diff
@@ -22,6 +22,18 @@ fn main() {
         .generate()
         .expect("Unable to generate bindings");
 
+    // Just tests that parsing <type_traits> doesn't crash, since we've have
+    // multiple hard-to-reduce regressions in that header.
```

Comment:
Aside: have you tried creducing the post-preprocessed test case? creduce can usually do a better job than trying to reduce manually.